polynote-frontend/polynote/state/notebook_state.ts

```diff
@@ -583,14 +583,15 @@ export class NotebookUpdateHandler extends Disposable { // extends ObjectStateHa
  */
 export function availableResultValues(symbols: KernelSymbols, cellOrder: number[], id?: number): Record<string, ResultValue> {
     const availableCells = Object.keys(symbols);
-    const whichCells = availableCells.filter(id => id.startsWith('-'));
+    // first, make sure to add any predef cells (they don't appear in cellOrder)
+    const cellsInScope = availableCells.filter(id => id.startsWith('-'));
```

Comment:
no changes here just commented and renamed some variables to clarify the implementation.